src/utils.js

```diff
@@ -34,7 +34,7 @@ function getStylusOptions(loaderContext, loaderOptions) {
       ? stylusOptions.resolveURL
       : { nocheck: true };
 
-  if (!stylusOptions.compress && isProductionLikeMode(loaderContext)) {
+  if (typeof stylusOptions.compress === 'undefined' && isProductionLikeMode(loaderContext)) {
```

Comment:
`(typeof stylusOptions.compress === 'undefined' || stylusOptions.compress) && isProductionLikeMode(loaderContext)`

Reply:
I am not sure what this comment is about :-)